content/content-strategy/manage-content-requests/create-content/index.yml

```diff
@@ -0,0 +1,18 @@
+pagetitle: Create content
+description: Start the process of creating content by guiding authors in how to write for plain English and web standards.
+weight: 30
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - toc.md
+  - why-head.md
+  - why.md
+  - who-head.md
+  - workflow-process.md
+  - tools-template-head.md
+  - tools-template.md
```

Comment:
Missing the global feedback partial, `- /_shared/feedback.md`. Please add at the bottom of the `main` section.

Reply:
Trisha.done